slices/aspnetcore-targeting-pack-8.0.yaml

```
@@ -0,0 +1,13 @@
+package: aspnetcore-targeting-pack-8.0
+
+essential:
+  - aspnetcore-targeting-pack-8.0_copyright
+
+slices:
+  libs:
+    contents:
```

```suggestion
    essential:
      - libc6_libs
      - libgcc-s1_libs
      - libssl3t64_libs
      - libstdc++6_libs
      - zlib1g_libs
    contents:
```

Copied from the 24.04 PR, but these look like the same dependencies showing on my 25.10 machine.

---

Oops, yes sorry - this was supposed to be on the apphost one :-)